pkgs/development/python-modules/rasterio/default.nix

```
@@ -1,22 +1,22 @@
 { buildPythonPackage, lib, fetchFromGitHub
 , cython
 , numpy, affine, attrs, cligj, click-plugins, snuggs, gdal
-, pytest, pytestcov, packaging, hypothesis, boto3
+, pytest, pytestcov, packaging, hypothesis, boto3, mock
 }:
 
 buildPythonPackage rec {
   pname = "rasterio";
-  version = "1.0.22";
+  version = "1.0.23";
 
   # Pypi doesn't ship the tests, so we fetch directly from GitHub
   src = fetchFromGitHub {
     owner = "mapbox";
     repo = "rasterio";
     rev = version;
-    sha256 = "1gx48qjiahlwflmjlkndp3ricd03jmzfx7i9ffgq7a2i6gcm36zp";
+    sha256 = "1xypc86prwkv6ypy87g6r1p5y6d6c51vfxd3z6hjd1hv2ks7zjs1";
   };
 
-  checkInputs = [ boto3 pytest pytestcov packaging hypothesis ];
+  checkInputs = [ boto3 pytest pytestcov packaging hypothesis ] ++ lib.optional (!isPy3k) [ mock ];
```

`lib.optional` second argument is a single element, I think you meant `optionals` here
```suggestion
  checkInputs = [ boto3 pytest pytestcov packaging hypothesis ] ++ lib.optionals (!isPy3k) [ mock ];
```

---

Indeed. Actually I meant not to use a list since there is only one element to add.